src/controllers/job-application.controller.ts

```diff
@@ -1,46 +1,49 @@
 import { Request, Response } from 'express';
-import { JobApplication } from '../models/job-application.model';
+import { applyForJobService, getJobApplicationsByJobIdService } from '../services/job-application.service';
 
 export const applyForJob = async (req: Request, res: Response) => {
 	try {
-		//Extract fields from request body
+		// Extract fields from request body
 		const { job_id, job_seeker_id, cover_letter, resume } = req.body;
```

Comment:
### Next time, Do not handle input validation this way.

Handle input validation at the route level using middleware. I created a validation middleware you can use to validate your request body. 

_Heres how you handle validation:_

route:
```
userProfileRoute.post('/profile/employer', authMiddleware, validateData(employerProfileSchema), createUserProfile);
```